content/projects/first-personal-website/_index.md

```diff
@@ -16,6 +16,11 @@ tags:
 - html
 - css
 title: Build your first personal website
+learning_outcomes:
+- code_hosting
+- code_responsive_page_design
+- code_web_page_structure
+- code_web_design_principals
```

Comment:
@Sbonelo01 the actual outcomes look good 👍🏾, but this won't work because these outcomes are expected to be in `learning_outcomes.yaml` but you've added them in a different file `pure_code_learning_outcomes.yaml`. So the stuff there needs to be moved to `learning_outcomes.yaml`; better if we keep them in the same place.